src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,175 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> first;
+
+    private class Node<E> {
+        private E item;
+        private Node<E> next;
+        private Node<E> prev;
+
+        Node(Node<E> prev, E element, Node<E> next) {
+            this.item = element;
+            this.next = next;
+            this.prev = prev;
+        }
+    }
+
     @Override
     public boolean add(T value) {
-        return false;
+        if (first == null) {
+            first = new Node<>(null, value, null);
+            return true;
+        }
+        Node<T> element = first;
+        while (element.next != null) {
+            element = element.next;
+        }
```

Comment:
Here you are looking for the last node, but when you have the field - Tail, you will already have this last node.